### PR TITLE
Special setting for hiding folders and paths from "About page"

### PR DESCRIPTION
**Pull request by maxkoryukov**
Special setting for hiding folders and paths from **About page**

Just add :

```ini

[core]
hide_about_dirs = 1

```

Thanks to **xvkdev**

### CODE REVIEW

**Comment by RuudBurger** [comment]
I'm missing the scss file changes.

**Comment by maxkoryukov** [comment]
There is no changes in the scss files.

File: couchpotato/static/scripts/combined.base.min.js [hunk 1]
```diff
@@ -1992,14 +1991,16 @@ var AboutSettingTab = new Class({
             self.createAbout();
         });
         self.settings.default_action = "about";
+        self.hideDirs = !!App.options && App.options["hide_about_dirs"];
```
Comment (maxkoryukov) [inline couchpotato/static/scripts/combined.base.min.js]: This file is generated by grunt and co, I did't change it

File: couchpotato/static/style/combined.min.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .page.movie_details,.thumb_list .movie.hover_start .actions{pointer-events:none}
 .movies>.description a:hover,.page.movie_details .releases .buttons a:hover{text-decoration:underline}
-.update.message{background:#FFF;padding:20px;text-align:center;font-size:1.25em}
+.update.message{background:#fff;padding:20px;text-align:center;font-size:1.25em}
```
Comment (maxkoryukov) [inline couchpotato/static/style/combined.min.css]: And this file I didn't change too.. Here are just lowercasing (my `grunt` make all **FFF** small, and I don't know why)

**Pull request merged**